src/common/transformations/src/transformations/common_optimizations/transpose_sinking_concat.cpp

```diff
@@ -71,6 +70,11 @@ ov::pass::TransposeSinkingConcatBackward::TransposeSinkingConcatBackward() {
         auto transpose = pattern_to_output.at(transpose_label).get_node_shared_ptr();
         auto main_node = pattern_to_output.at(main_node_label).get_node_shared_ptr();
 
+        if (main_node->output(0).get_target_inputs().size() > 1) {
+            auto new_node = CloneNodeWithoutConsumers(main_node, NodeVector{transpose});
```

Comment:
as I can see the current approach works this way
![image](https://user-images.githubusercontent.com/21181928/209328861-4534b1a2-f6ba-4542-b4ad-b62b67f40f99.png)
Using this approach we can expected 2x binary operations in the network (e.g. Node4 is binary and we need to move Transpose above one more time)

but I expected something like that
![image](https://user-images.githubusercontent.com/21181928/209328981-7b292ae7-bc37-4072-965f-935ffed11fc6.png)
But what we should do with these new Transposes (for Node1, Node2). I believe we need to discuss this

Reply:
Your approach leaves transpose nodes in network. I don't see how they will be moved than to start and end of network. I see my approach will decrease network performance but yours leaves transposes. GNA cannot work with transposes inside network. That should be discussed later. Just right now I can propose to implement the both strategies. Your strategy will be default. GNA will turn on my strategy. I can introduce flag as transformation constructor argument that switches strategies.

Reply:
@jane-intel , what do you think about it?

Reply:
@nkogteva, what do you think about it?

Reply:
I think we have to find any way to remove new added transposes otherwise we remove one output transpose and add new output transposes. Why we need such a transformation at all? What profit do we have with it?

Reply:
In general, I agree with @itikhono that transpose sinking should not lead to binary nodes duplication. @evkotov do you have real samples of topologies with such patterns which have to be run on GNA?